config/pipelines/high_throughput_scrna_core_cdna_prep.wip.yml

```diff
@@ -26,3 +26,24 @@ scRNA Core cDNA Prep Input:
     LRC HT 5p Chip: LRC HT 5p GEMs
     LRC HT 5p GEMs: LRC HT 5p cDNA PCR
     LRC HT 5p cDNA PCR: LRC HT 5p cDNA PCR XP
+scRNA Core Chromium HT Aggregation:
+  pipeline_group: scRNA Core cDNA Prep
+  filters:
+    request_type_key: limber_scrna_core_chromium_ht_aggregation
+  relationships:
+    # NB. alternate faculty input plate route included here
+    # TODO: do we need to allow skipping or bypassing the Aggregate plate? Config allows for a 2-step
+    # aggregation process, but this is not always necessary depending on the number of samples
+    # selected from the source plates. i.e. if 10 or fewer source plates are needed you could make the
+    # cherrypick plate directly from the source plates.
```

Comment:
Feels like this comment should be captured in a backlog story or similar, instead of in the code. It applies to the existing aggregation pipeline as well as this one.

Reply:
Yes, I can make one.
I recall a conversation with an SSR about the bespoke aggregation, and that they would create a cherrypick plate directly rather than use the aggregation pipeline if fewer sources were needed. So it might be that SS cherrypicking needs to allow picking from either LRC HT 5p cDNA Input or LRC HT 5p cDNA PCR XP plates into an LRC HT 5p Cherrypick directly.